docs/static/releasenotes.asciidoc

```diff
@@ -1,4 +1,94 @@
+* Remove `event_api.tags.illegal` for 9.0 https://github.com/elastic/logstash/pull/16461[#16461]
+* Remove deprecated modules netflow, fb_apache and azure https://github.com/elastic/logstash/pull/16514[#16514]
+* Remove ingest-converter https://github.com/elastic/logstash/pull/16453[#16453]
+* Drop JDK 11 support https://github.com/elastic/logstash/pull/16443[#16443]
+* Remove enterprise search from default distribution https://github.com/elastic/logstash/pull/16818[#16818]
+* Enable `log.format.json.fix_duplicate_message_fields` by default https://github.com/elastic/logstash/pull/16578[#16578]
+
+==== New features and enhancements
+* Add `%{{TIME_NOW}}` pattern for sprintf https://github.com/elastic/logstash/pull/16906[#16906]
+* Introduce a new `xpack.monitoring.allow_legacy_collection` flag to explicitly permit legacy monitoring https://github.com/elastic/logstash/pull/16586[#16586]
+* Use UBI as base image https://github.com/elastic/logstash/pull/16599[#16599]
+* Set `allow_superuser` to false as default https://github.com/elastic/logstash/pull/16558[#16558]
+
+==== Plugin releases
+
+*elastic_integration Filter - 9.0.0*
+
+* 9.0.0 prerelease https://github.com/elastic/logstash-filter-elastic_integration/pull/265[#265]
+
+*Elasticsearch Filter - 4.1.0*
+
+* Added support for custom headers https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/188[#188]
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/183[#183]
+
+*Http Filter - 2.0.0*
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-filter-http/pull/54[#54]
+
+*Beats Input - 7.0.1*
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-input-beats/pull/508[#508]
+
+*Elastic_serverless_forwarder Input - 2.0.0*
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-input-elastic_serverless_forwarder/pull/11[#11]
+
+* Promote from technical preview to GA https://github.com/logstash-plugins/logstash-input-elastic_serverless_forwarder/pull/10[#10]
+
+*Elasticsearch Input - 5.0.0*
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-input-elasticsearch/pull/213[#213]
+
+*Http Input - 4.1.0*
+
+* Add improved proactive rate-limiting, rejecting new requests when queue has been actively blocking for more than 10 seconds https://github.com/logstash-plugins/logstash-input-http/pull/186[#186]
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-input-http/pull/182[#182]
+
+*Http_poller Input - 6.0.0*
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-input-http_poller/pull/149[#149]
+
+*Tcp Input - 7.0.0*
+
+* Remove deprecated SSL settings https://github.com/logstash-plugins/logstash-input-tcp/pull/228[#228]
+
+*Elastic_enterprise_search Integration - 3.0.1*
```

Comment:
Is this necessary?

Reply:
It is part of enterprise search (mentioned in breaking change section), I will remove this.